src/main/java/com/google/sps/data/Card.java

```diff
@@ -104,7 +103,7 @@ public Entity createEntity(Key folderKey) {
         return card;
     }
```

Comment:
How did this code get into the main branch? I don't see a pull request for it.

Reply:
Yeah my bad. I think there was a miscommunication between the podmates. I didn't know Setup was the main branch on that day, so I ended up pushing some code. Currently the pushed code that you and Zach haven't gotten the chance to review is Folder.java, Card.java, User.java.